src/configs/config-fonts.json

```
@@ -0,0 +1,35 @@
+{
+  "source": ["src/tokens/design-tokens.json", "src/tokens/design-tokens-theme-pro.json"],
```

s'il est commun, il ne faut pas qu'il y ai des trucs de pro

```suggestion
  "source": ["src/tokens/design-tokens.json"],
```

---

bien vu merci